### ISSUE TITLE
npm install issues

### ISSUE THREAD

**Issue by sghiassy** [chilts/node-ofx#5]
Looks like installing the package via npm doesn't work. Any ideas?

```
└─▪ npm install ofx --save

> node-expat@2.0.0 install /Users/sghiassy/Desktop/test/node_modules/node-expat
> node-gyp rebuild

gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
  CC(target) Release/obj.target/expat/deps/libexpat/lib/xmlparse.o
  CC(target) Release/obj.target/expat/deps/libexpat/lib/xmltok.o
../deps/libexpat/lib/xmltok.c:471:1: warning: missing field 'isName2' initializer [-Wmissing-field-initializers]
};
^
../deps/libexpat/lib/xmltok.c:484:1: warning: missing field 'isName2' initializer [-Wmissing-field-initializers]
};
^
../deps/libexpat/lib/xmltok.c:504:1: warning: missing field 'isName2' initializer [-Wmissing-field-initializers]
};
^
../deps/libexpat/lib/xmltok.c:517:1: warning: missing field 'isName2' initializer [-Wmissing-field-initializers]
};
^
../deps/libexpat/lib/xmltok.c:730:1: warning: missing field 'isName2' initializer [-Wmissing-field-initializers]
};
^
../deps/libexpat/lib/xmltok.c:749:1: warning: missing field 'isName2' initializer [-Wmissing-field-initializers]
};
^
../deps/libexpat/lib/xmltok.c:762:1: warning: missing field 'isName2' initializer [-Wmissing-field-initializers]
};
^
../deps/libexpat/lib/xmltok.c:775:1: warning: missing field 'isName2' initializer [-Wmissing-field-initializers]
};
^
../deps/libexpat/lib/xmltok.c:871:1: warning: missing field 'isName2' initializer [-Wmissing-field-initializers]
};
^
../deps/libexpat/lib/xmltok.c:890:1: warning: missing field 'isName2' initializer [-Wmissing-field-initializers]
};
^
10 warnings generated.
  CC(target) Release/obj.target/expat/deps/libexpat/lib/xmlrole.o
  LIBTOOL-STATIC Release/libexpat.a
  CXX(target) Release/obj.target/node_expat/node-expat.o
../node-expat.cc:54:34: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> New(const Arguments& args)
                                 ^~~~~~~~~
                                 v8::internal::Arguments
/Users/sghiassy/.node-gyp/6.9.1/include/node/v8.h:147:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
../node-expat.cc:99:36: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> Parse(const Arguments& args)
                                   ^~~~~~~~~
                                   v8::internal::Arguments
/Users/sghiassy/.node-gyp/6.9.1/include/node/v8.h:147:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
../node-expat.cc:171:42: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> SetEncoding(const Arguments& args)
                                         ^~~~~~~~~
                                         v8::internal::Arguments
/Users/sghiassy/.node-gyp/6.9.1/include/node/v8.h:147:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
../node-expat.cc:198:39: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> GetError(const Arguments& args)
                                      ^~~~~~~~~
                                      v8::internal::Arguments
/Users/sghiassy/.node-gyp/6.9.1/include/node/v8.h:147:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
../node-expat.cc:212:35: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> Stop(const Arguments& args)
                                  ^~~~~~~~~
                                  v8::internal::Arguments
/Users/sghiassy/.node-gyp/6.9.1/include/node/v8.h:147:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
../node-expat.cc:229:37: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> Resume(const Arguments& args)
                                    ^~~~~~~~~
                                    v8::internal::Arguments
/Users/sghiassy/.node-gyp/6.9.1/include/node/v8.h:147:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
../node-expat.cc:244:36: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> Reset(const Arguments& args)
                                   ^~~~~~~~~
                                   v8::internal::Arguments
/Users/sghiassy/.node-gyp/6.9.1/include/node/v8.h:147:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
../node-expat.cc:272:51: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> GetCurrentLineNumber(const Arguments& args)
                                                  ^~~~~~~~~
                                                  v8::internal::Arguments
/Users/sghiassy/.node-gyp/6.9.1/include/node/v8.h:147:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
../node-expat.cc:285:53: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> GetCurrentColumnNumber(const Arguments& args)
                                                    ^~~~~~~~~
                                                    v8::internal::Arguments
/Users/sghiassy/.node-gyp/6.9.1/include/node/v8.h:147:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
../node-expat.cc:298:50: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> GetCurrentByteIndex(const Arguments& args)
                                                 ^~~~~~~~~
                                                 v8::internal::Arguments
/Users/sghiassy/.node-gyp/6.9.1/include/node/v8.h:147:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
../node-expat.cc:22:17: error: calling a protected constructor of class 'v8::HandleScope'
    HandleScope scope;
                ^
/Users/sghiassy/.node-gyp/6.9.1/include/node/v8.h:915:13: note: declared protected here
  V8_INLINE HandleScope() {}
            ^
../node-expat.cc:23:33: error: no matching function for call to 'New'
    Local<FunctionTemplate> t = FunctionTemplate::New(New);
                                ^~~~~~~~~~~~~~~~~~~~~
/Users/sghiassy/.node-gyp/6.9.1/include/node/v8.h:4483:34: note: candidate function not viable: no known conversion from 'Handle<v8::Value> (const v8::internal::Arguments &)' (aka
      'v8::Local<v8::Value> (const v8::internal::Arguments &)') to 'v8::Isolate *' for 1st argument
  static Local<FunctionTemplate> New(
                                 ^
/Users/sghiassy/.node-gyp/6.9.1/include/node/v8.h:4487:34: note: candidate function not viable: requires 6 arguments, but 1 was provided
  static Local<FunctionTemplate> New(
                                 ^
../node-expat.cc:27:43: error: cannot initialize a parameter of type 'v8::FunctionCallback' (aka 'void (*)(const FunctionCallbackInfo<v8::Value> &)') with an lvalue of type
      'Handle<v8::Value> (const v8::internal::Arguments &)' (aka 'v8::Local<v8::Value> (const v8::internal::Arguments &)'): type mismatch at 1st parameter
      ('const FunctionCallbackInfo<v8::Value> &' vs 'const v8::internal::Arguments &')
    NODE_SET_PROTOTYPE_METHOD(t, "parse", Parse);
                                          ^~~~~
/Users/sghiassy/.node-gyp/6.9.1/include/node/node.h:275:60: note: passing argument to parameter 'callback' here
                                      v8::FunctionCallback callback) {
                                                           ^
../node-expat.cc:28:49: error: cannot initialize a parameter of type 'v8::FunctionCallback' (aka 'void (*)(const FunctionCallbackInfo<v8::Value> &)') with an lvalue of type
      'Handle<v8::Value> (const v8::internal::Arguments &)' (aka 'v8::Local<v8::Value> (const v8::internal::Arguments &)'): type mismatch at 1st parameter
      ('const FunctionCallbackInfo<v8::Value> &' vs 'const v8::internal::Arguments &')
    NODE_SET_PROTOTYPE_METHOD(t, "setEncoding", SetEncoding);
                                                ^~~~~~~~~~~
/Users/sghiassy/.node-gyp/6.9.1/include/node/node.h:275:60: note: passing argument to parameter 'callback' here
                                      v8::FunctionCallback callback) {
                                                           ^
../node-expat.cc:29:46: error: cannot initialize a parameter of type 'v8::FunctionCallback' (aka 'void (*)(const FunctionCallbackInfo<v8::Value> &)') with an lvalue of type
      'Handle<v8::Value> (const v8::internal::Arguments &)' (aka 'v8::Local<v8::Value> (const v8::internal::Arguments &)'): type mismatch at 1st parameter
      ('const FunctionCallbackInfo<v8::Value> &' vs 'const v8::internal::Arguments &')
    NODE_SET_PROTOTYPE_METHOD(t, "getError", GetError);
                                             ^~~~~~~~
/Users/sghiassy/.node-gyp/6.9.1/include/node/node.h:275:60: note: passing argument to parameter 'callback' here
                                      v8::FunctionCallback callback) {
                                                           ^
../node-expat.cc:30:42: error: cannot initialize a parameter of type 'v8::FunctionCallback' (aka 'void (*)(const FunctionCallbackInfo<v8::Value> &)') with an lvalue of type
      'Handle<v8::Value> (const v8::internal::Arguments &)' (aka 'v8::Local<v8::Value> (const v8::internal::Arguments &)'): type mismatch at 1st parameter
      ('const FunctionCallbackInfo<v8::Value> &' vs 'const v8::internal::Arguments &')
    NODE_SET_PROTOTYPE_METHOD(t, "stop", Stop);
                                         ^~~~
/Users/sghiassy/.node-gyp/6.9.1/include/node/node.h:275:60: note: passing argument to parameter 'callback' here
                                      v8::FunctionCallback callback) {
                                                           ^
../node-expat.cc:31:44: error: cannot initialize a parameter of type 'v8::FunctionCallback' (aka 'void (*)(const FunctionCallbackInfo<v8::Value> &)') with an lvalue of type
      'Handle<v8::Value> (const v8::internal::Arguments &)' (aka 'v8::Local<v8::Value> (const v8::internal::Arguments &)'): type mismatch at 1st parameter
      ('const FunctionCallbackInfo<v8::Value> &' vs 'const v8::internal::Arguments &')
    NODE_SET_PROTOTYPE_METHOD(t, "resume", Resume);
                                           ^~~~~~
/Users/sghiassy/.node-gyp/6.9.1/include/node/node.h:275:60: note: passing argument to parameter 'callback' here
                                      v8::FunctionCallback callback) {
                                                           ^
../node-expat.cc:32:43: error: cannot initialize a parameter of type 'v8::FunctionCallback' (aka 'void (*)(const FunctionCallbackInfo<v8::Value> &)') with an lvalue of type
      'Handle<v8::Value> (const v8::internal::Arguments &)' (aka 'v8::Local<v8::Value> (const v8::internal::Arguments &)'): type mismatch at 1st parameter
      ('const FunctionCallbackInfo<v8::Value> &' vs 'const v8::internal::Arguments &')
    NODE_SET_PROTOTYPE_METHOD(t, "reset", Reset);
                                          ^~~~~
/Users/sghiassy/.node-gyp/6.9.1/include/node/node.h:275:60: note: passing argument to parameter 'callback' here
                                      v8::FunctionCallback callback) {
                                                           ^
../node-expat.cc:33:58: error: cannot initialize a parameter of type 'v8::FunctionCallback' (aka 'void (*)(const FunctionCallbackInfo<v8::Value> &)') with an lvalue of type
      'Handle<v8::Value> (const v8::internal::Arguments &)' (aka 'v8::Local<v8::Value> (const v8::internal::Arguments &)'): type mismatch at 1st parameter
      ('const FunctionCallbackInfo<v8::Value> &' vs 'const v8::internal::Arguments &')
    NODE_SET_PROTOTYPE_METHOD(t, "getCurrentLineNumber", GetCurrentLineNumber);
                                                         ^~~~~~~~~~~~~~~~~~~~
/Users/sghiassy/.node-gyp/6.9.1/include/node/node.h:275:60: note: passing argument to parameter 'callback' here
                                      v8::FunctionCallback callback) {
                                                           ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
make: *** [Release/obj.target/node_expat/node-expat.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/sghiassy/.nvm/versions/node/v6.9.1/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
gyp ERR! System Darwin 16.1.0
gyp ERR! command "/Users/sghiassy/.nvm/versions/node/v6.9.1/bin/node" "/Users/sghiassy/.nvm/versions/node/v6.9.1/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/sghiassy/Desktop/test/node_modules/node-expat
gyp ERR! node -v v6.9.1
gyp ERR! node-gyp -v v3.4.0
gyp ERR! not ok
npm WARN test@1.0.0 No description
npm WARN test@1.0.0 No repository field.
npm ERR! Darwin 16.1.0
npm ERR! argv "/Users/sghiassy/.nvm/versions/node/v6.9.1/bin/node" "/Users/sghiassy/.nvm/versions/node/v6.9.1/bin/npm" "install" "ofx" "--save"
npm ERR! node v6.9.1
npm ERR! npm  v3.10.8
npm ERR! code ELIFECYCLE

npm ERR! node-expat@2.0.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the node-expat@2.0.0 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the node-expat package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs node-expat
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls node-expat
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/sghiassy/Desktop/test/npm-debug.log
```

**Comment by chilts** [comment]
Something in `xml2json` was failing, so I've updated the dependencies and it seems to work. Released as v0.3.0 to npm. Let me know if it's all okay for you or something still fails. Thanks for opening.

**Comment by sghiassy** [comment]
Works now. Thanks!
